### DISCUSSION TITLE
quinn lacks support for QUIC multipath extension

### DISCUSSION THREAD

**Discussion by dvasanth** [libp2p/rust-libp2p#5612] [Ideas]
I am looking to add multipath support for QUIC in libp2p transport. Quinn implementation is tightly coupled with libp2p transport. It lacks support for multipath. Any body have tired using other implementation of quic like TQUIC in place of quinn?

**Comment by drHuangMHT** [comment]
Similar issue: #5608. Anything outside of [specs](https://github.com/libp2p/specs/tree/master/quic) will be deemed low priority, although contributions welcome!
